images/chaos-daemon/Dockerfile

```diff
@@ -6,9 +6,9 @@ ARG HTTP_PROXY
 ENV http_proxy $HTTP_PROXY
 ENV https_proxy $HTTPS_PROXY
 
-RUN apt-get update && \
+RUN apt-get update && \ 
     apt-get install -y tzdata iptables ipset stress-ng iproute2 fuse util-linux procps curl && \
-    rm -rf /var/lib/apt/lists/*
+    apt-get clean
```

Comment:
we could not use `apt-get clean` instead of `rm -rf /var/lib/apt/lists/*` for shrinking image size. part fo `man apt-get`

```
clean
    clean clears out the local repository of retrieved package files. It removes everything but the lock file from /var/cache/apt/archives/ and /var/cache/apt/archives/partial/.

```

Reply:
:+1: . I will remove all `apt-get clean` and replace with `rm -rf /var/lib/apt/lists/*`